generate/generate.go

```diff
@@ -456,21 +486,44 @@ func (g *Generator) ClearProcessEnv() {
 		return
 	}
 	g.Config.Process.Env = []string{}
+	// Clear out the env cache map as well
+	g.envMap = make(map[string]int)
```

Comment:
would `nil` accomplish this as well?

Reply:
I got rid of the make, and just define the map. I think nil might cause issues, but hopefully this is better.